tests/test_nsjail.py

```diff
@@ -272,14 +336,14 @@ def test_large_output_is_truncated(self):
         self.assertEqual(output, chunk * expected_chunks)
 
     def test_nsjail_args(self):
-        args = ("foo", "bar")
+        args = ["foo", "bar"]
```

Comment:
It's due to the fact I've changed the call within the function to a list and not a tuple since I had to do a couple extend/appends. Wasn't sure whether to cast back to a tuple or just leave it and ended up just leaving it. Do you think it's better if we cast it to a tuple at the end?

Reply:
Thanks for clarifying. No objections to keeping it a list.